### PR TITLE
Db create static pages

### PR DESCRIPTION
**Pull request by benkt2**
Added some more static pages, a manky menu and fixed db:create

### CODE REVIEW

File: db/migrate/20180911103950_create_join_table_ecosystem_services_assets.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class CreateJoinTableEcosystemServicesAssets < ActiveRecord::Migration[5.2]
   def change
     create_join_table :ecosystem_services, :assets do |t|
-      t.index [:ecosystem_service_id, :asset_id]
-      t.index [:asset_id, :ecosystem_service_id]
+      t.index([:ecosystem_service_id, :asset_id], name: 'idx_ecosystem_service_asset')
+      t.index([:asset_id, :ecosystem_service_id], name: 'idx__asset_ecosystem_service')
```
Comment (spencerldixon) [inline db/migrate/20180911103950_create_join_table_ecosystem_services_assets.rb]: What happened here?

Comment (spencerldixon) [inline db/migrate/20180911103950_create_join_table_ecosystem_services_assets.rb]: What happened here?

Comment (benkt2) [reply db/migrate/20180911103950_create_join_table_ecosystem_services_assets.rb]: The rails generated index name was something like ecosystem_services_assets_index_ecosystem_service_id_asset_id 
and it was longer than Postgres's 63 char limit so failed on db:create (at least on my version of Postgres) so I added a specific short-named index which is hopefully nearly as descriptive

Comment (spencerldixon) [reply db/migrate/20180911103950_create_join_table_ecosystem_services_assets.rb]: Nice! Thanks @benkt2

**Pull request merged**